codes/kotlin/chapter_sorting/QuickSort.kt

```diff
@@ -0,0 +1,37 @@
+/**
+ * File: QuickSort.kt
+ * Created Time: 2024-1-25
+ * Author: curtishd (1023632660@qq.com)
+ */
+
+package chapter_sorting.QuickSort
+
+/* 元素交换 */
+fun swap(nums: IntArray, i: Int, j: Int) {
+    nums[i] = nums[j].also { nums[j] = nums[i] }
```

Comment:
Could you put all the quick sort functions into one file refer to other languages?

Reply:
Sure！